wweval/R/post_processing.R

```diff
@@ -1,48 +1,27 @@
 #' Get model draws combined with input and evaluation data
 #'
+#' @param fit_obj_wwinference wwinference_fit object that is returned when
+#' wwinference::wwinference() is run.
 #' @param model_output the type of model expected observation you want,
 #' options are "hosp" and "ww"
 #' @param model_type The type of model, options are "ww" and "hosp"
-#' @param draws The raw draws dataframe
 #' @param forecast_date The date the forecast was made
 #' @param scenario A name for the scenario that the input
 #' data represents, as a string.
 #' @param location The location for which the model is being run.
-#' @param input_data The input dataset used for fitting the model.
 #' @param eval_data The retrospective dataset used to evaluate the model
 #' (should have data beyond the forecast date)
-#' @param last_hosp_data_date The date of the last hospital admissions data point
-#' that the model is calibrated to
-#' @param ot An integer indicating the number of days the model is calibrated
-#' to hospital admissions data
-#' @param forecast_time  An integer indicate the time in days of the forecast
-#'
 #' @return a dataframe of model draws subsetted to only the specified output
 #' type, joined with the evaluation data and the input calibration data
 #' @export
-get_model_draws_w_data <- function(model_output,
-                                   model_type = c("ww", "hosp"),
-                                   draws,
-                                   forecast_date,
-                                   scenario,
-                                   location,
-                                   input_data,
-                                   eval_data,
-                                   last_hosp_data_date,
-                                   ot,
-                                   forecast_time = 28) {
+new_get_model_draws_w_data <- function(fit_obj_wwinference,
```

Comment:
any reason not to name this the same as the old function?

Reply:
I can change it! Was just comparing while deving